app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -258,6 +261,11 @@ class ConversationViewModel @Inject constructor(
         conversationViewState = conversationViewState.copy(downloadedAssetDialogState = Hidden)
     }
 
+    suspend fun isFileSharingEnabled(): Boolean = withContext(Dispatchers.IO) {
+        isFileSharingEnabled.invoke()
```

Comment:
what is this supposed to do ? the MVI flow would be : fire a use case > update the state object > let the UI handle state change